java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileInfoStore.java

```diff
@@ -435,6 +435,11 @@ public Map<String, List<String>> getPartitionToActiveFilesMap() throws StateStor
         return partitionToFiles;
     }
 
+    @Override
+    public void initialise() throws StateStoreException {
+
+    }
```

Comment:
Very minor, but I think in other places in the codebase, if a method has no content then the closing } goes on the next line - there's no blank line.